env/dev.env

```
@@ -8,8 +8,9 @@
 PORT=3000
 JWT_SECRET="e1d187fdc6a4ad4929399120db12ac0ebc72b66228a0b463ccd8a218b68fc83c"
 URL_WALLET="http://localhost:8000"
+URL_RSA="https://udmeo4gmr7lqazfhhv7u2si7ce0ahdsi.lambda-url.us-east-1.on.aws"
```

Esse URL é o antigo. Não funciona mais.

Agora entendi o que você quis dizer de por a API aqui. Então, por mais que não tenha problema colocar, é melhor não colocar porque como o repositório é público dá pra fazer um ataque DDoS usando a URL, sabe? Porque nossa API não tem autenticação por token ou qualquer outro tipo de autenticação (o que seria o ideal, mas pelo tempo que temos não vai rolar agora).

Vou deixar uma sugestão de mudança pro endereço local.

```suggestion
URL_RSA="http://localhost:9000"
```

---

Bele, vou remover aqui